local-cli/generator-android/templates/package/MainActivity.java

```diff
@@ -25,7 +25,7 @@ protected void onCreate(Bundle savedInstanceState) {
                 .setApplication(getApplication())
                 .setBundleAssetName("index.android.bundle")
                 .setJSMainModuleName("index.android")
-                .addPackage(new MainReactPackage())
+                .addPackage(new MainReactPackage(this))
```

Comment:
We need to keep the current constructor around too for backwards-compatibility, otherwise apps won't compile after upgrading.

Have an idea how to get rid of this issue by introducing a base Activity for React apps. Stay tuned :)